Polar/inputs-collect/download-sources.py

```diff
@@ -0,0 +1,41 @@
+import json
+import requests
+import tarfile
+import zipfile
+import getpass
+import os
+from tqdm import tqdm
+
+pathOut = "~/data/polar/source/"
+pathOut = os.path.expanduser(pathOut)
+
+if not os.path.exists(pathOut):
+    os.makedirs(pathOut)
+    
+with open('../cube.geojson', 'r') as f:
```

Comment:
```suggestion
with open('cube.geojson', 'r') as f:
```

The readme assumes the user is in the "Polar" directory, therefore the script does not work, asl cube.geojson is on the same level as the readme.

Reply:
so either adjust the code, or change the readme :)